src/riak_kv_qry.erl

```diff
@@ -202,6 +202,8 @@ make_insert_row([], _Positions, Row, _FieldTypes) ->
     %% Out of entries in the value - row is populated with default values
     %% so if we run out of data for implicit/explicit fieldnames can just return
     {ok, Row};
+make_insert_row([{identifier, _Identifier} | _Values], _Positions, _Row, _FieldTypes) ->
+    {error, identifier_unexpected};
```

Comment:
Make sure this error has a human-readable message. It's a matter of adding another `make_errorthing_resp` and adding a case branch here: https://github.com/basho/riak_kv/blob/riak_ts-develop/src/riak_kv_ts_svc.erl#L455L461.